test/transformer/descriptor/utils/namespace/namespace.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+var NameSpaceInterfaceImport;
+(function (NameSpaceInterfaceImport) {
+    var Enum;
```

Comment:
Why is this file pushed?

Reply:
Ahah by mistake, now im using git add by commandline and sometimes I forget to exclude the one I dont want! Ill remove it :)

Reply:
Removed